runtime/wasm/src/host_exports.rs

```diff
@@ -560,6 +560,44 @@ impl HostExports {
         Ok(x.pow(exponent))
     }
 
+    pub(crate) fn big_int_from_string(&self, s: String) -> Result<BigInt, DeterministicHostError> {
+        BigInt::from_str(&s)
+            .with_context(|| format!("string  is not a BigInt: '{}'", s))
```

Comment:
There's a double space after "string". And I think our convention is to wrap `{}` in backticks rather than single quotes, like here: https://github.com/graphprotocol/graph-node/pull/2151/files#diff-9f6f2b4b1457abc5d61a58aa2ba3f2b3a2a4f2010def91f76dca50d4780c2f38L547

Reply:
Fixed!